src/PageContent/Settings/AccountManager.js

```diff
@@ -0,0 +1,163 @@
+
+        return (
+            <div className="scs-0">
+                <div className="scs-0-content scs-1-content">
+                    <h5>Account Manager Details</h5>
+
+                    {!this.props.isEditingAcc ? 
+                        
+                        <div>
+                           
+                            <div className="amd-row">
+                                <div className="amd-details amd-details-1">
+                                    <div className="amd-details-child">
+                                        <h6>{this.props.account.details.name}</h6>
+                                        <h6>{this.props.account.details.position}</h6>
+                                        <h6>{this.props.account.details.email}</h6>
+                                        <h6>{this.props.account.details.phone}</h6>
+                                    </div>
+                                    
+                                    <div className="amd-details-child">
+                                        <h6>Recieve Notifications via</h6>
+                                        <h6>Feature coming soon!</h6>
+                                        {/* {this.props.account.smsNotif ? <h6>SMS/Text Message</h6> : <div/>}
+                                        {this.props.account.emailNotif ? <h6>Email</h6> : <div/>} */}
+                                    </div>
+                                </div>
+                            </div>
+
+                            {this.props.secondaryContact ?
+                                <div className="amd-row">
+                                    <div className="amd-details amd-details-1">
+                                        <div className="amd-details-child">
+                                            <h6>{this.props.secondaryContact.name}</h6>
+                                            <h6>{this.props.secondaryContact.position}</h6>
+                                            <h6>{this.props.secondaryContact.email}</h6>
+                                            <h6>{this.props.secondaryContact.phone}</h6>
+                                        </div>
+                                        
+                                        <div className="amd-details-child">
+                                            <h6>Recieve Notifications via</h6>
+                                            <h6>Feature coming soon!</h6>
+                                            {/* {this.props.account.smsNotif ? <h6>SMS/Text Message</h6> : <div/>}
+                                            {this.props.account.emailNotif ? <h6>Email</h6> : <div/>} */}
+                                        </div>
+                                    </div>
+                                </div>
+                                :
+                                <span />
+                            }
+
+                            {this.props.isAdmin === 'admin' ? 
```

Comment:
same thing about RA and DGs wont have `isAdmin`. See comment in `OrganizationDetails`

Reply:
Fixed.